File: VB/Person.vb
Imports Microsoft.VisualBasic
Imports System

Namespace TypeConverterTest
	Public Class Person
		Private name_ As String
		Private address_ As Address

		Public Sub New(ByVal name As String, ByVal address As Address)
			name_ = name
			address_ = address
		End Sub

		Public Property Name() As String
			Get
				Return name_
			End Get
			Set(ByVal value As String)
				name_ = value
			End Set
		End Property

		Public Property Address() As Address
			Get
				Return address_
			End Get
			Set(ByVal value As Address)
				address_ = value
			End Set
		End Property
	End Class
End Namespace

File: CS/Person.cs
using System;

namespace TypeConverterTest {
	public class Person {
		private string name_;
		private Address address_;

		public Person(string name, Address address) {
			name_ = name;
			address_ = address;
		}

		public string Name {
			get {
				return name_;
			}
			set {
				name_ = value;
			}
		}

		public Address Address {
			get {
				return address_;
			}
			set {
				address_ = value;
			}
		}
	}
}
